hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/feature/index/TestExpressionIndex.scala

```diff
@@ -1776,7 +1776,7 @@ class TestExpressionIndex extends HoodieSparkSqlTestBase {
              |""".stripMargin)
 
         // create index using bloom filters on city column with upper() function
-        spark.sql(s"create index idx_bloom_$tableName on $tableName using bloom_filters(city) options(expr='upper')")
+        spark.sql(s"create index idx_bloom_$tableName on $tableName using bloom_filters(city) options(expr='upper', fpp='0.01')")
```

Comment:
Should we test all options?  Could you add validation on these params so that they are stored in the index definition and actually take effect?

Reply:
Sure. Now i added all options and also validating that in index metadata.